client/scripts/sw-docker-entrypoint

```diff
@@ -39,6 +39,7 @@ pre_config() {
 [global]
 index-url = ${SW_PYPI_INDEX_URL}
 extra-index-url = ${SW_PYPI_EXTRA_INDEX_URL}
+timeout = ${SW_PYPI_TIMEOUT:-90}
```

Comment:
Is there has a ‘RETRIES’ config? and add it for the following command of `python3 -m pip install "starwhale==${DETECTED_SW_VER}"`

Reply:
pip.conf does not include retries config.

Reply:
ok